install/minthcm-install.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2025 minthcm
+# Author: MintHCM
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://github.com/minthcm/minthcm
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Setting up PHP 8.2"
+PHP_APACHE="YES" PHP_VERSION="8.2" PHP_MODULE="mysql,cli,redis" PHP_FPM="YES" setup_php
+msg_ok "Set up PHP 8.2"
+
+msg_info "Setting up Composer"
+setup_composer
+msg_ok "Set up Composer"
+
```

Comment:
```suggestion
setup_composer

```
Func already has msg block